ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__amd64-stable-4.15-upgrade-from-stable-4.10.yaml

```diff
@@ -104,7 +104,6 @@ tests:
           cpu: 100m
           memory: 200Mi
     - ref: cucushift-chainupgrade-toimage
-    - chain: openshift-upgrade-qe-sanity
```

Comment:
should be replaced with `openshift-upgrade-qe-test-chainupgrade`?

Reply:
this is a stable to stable chain upgrade, do not need any e2e testing, while `openshift-upgrade-qe-test-chainupgrade` is for sable to nightly chain upgrade, which includes e2e testing. So this is expected.